Readme.md

```diff
@@ -0,0 +1,56 @@
+Answers to assignment 1 of week 2
+---------------------------------
+
+a) The two size classes in iOS are classes for the horizontal and vertical size of devices. In portrait mode it is .regular vertically and .compact horizontally, while both are .compact in landscape mode.
+
+b) Continuous Learning means to always going on in learning and never be finished. This is important in mobile dev since the base technologies evolve (eg Swift in version 5.9 has many new features), new technologies enter the scene (SwiftUI, Combine, SwiftData) and there are many frameworks and tools around which also evolve (CI/CD, DevOps, Cloud).
+
+c) To find the modifiers of a particular view in the UI: Look in the source code. I remember from the videos, that one can set the canvas in Xcode to selectable and then select elements of the UI in the canvas and find, where they are in the code editor, but I cannot find info about this, and if I try this out, it does not work with simple clicking, cmd-clicking, alt-clicking or ctrl-clicking. So I'm clueless here.
+
```

Comment:
When you are looking at a preview and your code, you can open the Attributes Inspector (option-command-4) on the right. Then when you click on elements in the code, the modifiers will appear. You can also add and view modifiers in that inspector and the code will update. To make the Canvas selectable, first click on the pointer icon
![image](https://github.com/akoelbl/week2-solution/assets/326770/112bc430-49f5-41f4-b200-b231e3c4bd1c)
then you can command click on an element to show it in code and to show it's attributes in the attribute inspector. Finally, you can simply type "." in code at the end of the element and all of the available modifiers for that element will appear in the autocomplete dropdown.